### PR TITLE
ndk/bitmap: Gate `BitmapCompressError` behind `api-level-30`

### PR DESCRIPTION
**Pull request by Rodrigodd**
Otherwise there is a compilation error when building with only the feature `bitmap`, due to the inexistence of the `DataSpace` type. `DataSpace` is defined with `api-level-28`, but this `BitmapCompressError` is only used by `Bitmap::compress()` and `Bitmap::compress_raw()`, which are only available with `api-level-30`.

### CODE REVIEW

**Comment by MarijnS95** [comment]
Well that is fair even though this will all be deleted again soon. We cannot merge, yank, and re-release this without a changelog entry though.

**Comment by Rodrigodd** [comment]
@MarijnS95 Added a changelog entry.

File: ndk/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 - Move `MediaFormat` from `media::media_codec` to its own `media::media_format` module. (#442)
 - media_format: Expose `MediaFormat::copy()` and `MediaFormat::clear()` from API level 29. (#449)
+- ndk/bitmap: Fix missing `cfg(feature = "api-level-30")` on `BitmapCompressError` (#462)
```
Comment (MarijnS95) [inline ndk/CHANGELOG.md:5]:
```suggestion
- ndk/bitmap: Guard `BitmapCompressError` behind missing `api-level-30` feature. (#462)
```

**Review by MarijnS95 — APPROVED** [review]

File: ndk/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 - Move `MediaFormat` from `media::media_codec` to its own `media::media_format` module. (#442)
 - media_format: Expose `MediaFormat::copy()` and `MediaFormat::clear()` from API level 29. (#449)
+- ndk/bitmap: Guard `BitmapCompressError` behind missing `api-level-30` feature. (#462)
```
Comment (MarijnS95) [inline ndk/CHANGELOG.md:5]:
```suggestion
- bitmap: Guard `BitmapCompressError` behind missing `api-level-30` feature. (#462)
```

This is already the NDK-specific changelog.

**Pull request merged**